### PR TITLE
Update dependencies, target latest framework

### PR DESCRIPTION
**Pull request by barnhill**
Updates to the latest of the dependencies at the time of this PR.  Also removes the use of build tools that are now incorporated into the gradle plugin by default.

### CODE REVIEW

File: sample/build.gradle [hunk 1]
```diff
@@ -5,8 +5,7 @@ repositories {
 }
 
 android {
-    compileSdkVersion 28
-    buildToolsVersion "28.0.3"
```
Comment (barnhill) [inline sample/build.gradle:9]: this is no longer necessary due to the gradle plugin incorporating this by default and using the latest

**Review by benjamin-bader — APPROVED** [review]
LGTM, thanks for splitting these PRs apart!

**Comment by barnhill** [comment]
Ready to merge?

**Comment by emarc-m** [comment]
Thank you @barnhill! Kindly add me as a reviewer as well in future PRs.

**Pull request merged**